book/src/gobject_properties.md

```diff
@@ -39,21 +52,26 @@ Whenever we now click on one of the two switches, the other one gets toggled as
 
 We can also add properties to custom GObjects.
 We can demonstrate that, by binding the `number` of our `CustomButton` to a property.
+For that, we need to be able to [lazily evaluate](https://en.wikipedia.org/wiki/Lazy_evaluation) expressions.
+`once_cell` does that for us and is already part of `std` of Rust nightly.
```

Comment:
It provides us with many other types too. The one we use here is called `Lazy` :)